python/cudf/cudf/_lib/nvtx/nvtx.py

```diff
@@ -0,0 +1,114 @@
+from contextlib import contextmanager
+
+import nvtx._lib as libnvtx
```

Comment:
If we want this to be as performant as possible we should explicitly import what we need to avoid unnecessary lookups into the module.